lib/streamlit/app_session.py

```diff
@@ -129,8 +124,21 @@ def __init__(
 
         self._session_state = SessionState()
 
+        self.register_change_listeners()
+
         LOGGER.debug("AppSession initialized (id=%s)", self.id)
 
+    def register_change_listeners(self):
```

Comment:
Can we add types to any new functions (so like, just add a `-> None` to this one) to keep pushing towards 100% typed?